restapi/src/test/java/io/stargate/web/docsapi/service/query/search/resolver/filter/impl/InMemoryCandidatesFilterTest.java

```diff
@@ -120,12 +123,13 @@ public void fixedPath() throws Exception {
 
       withQuery(
           TABLE,
-          "SELECT key, leaf, text_value, dbl_value, bool_value, p0, p1, WRITETIME(leaf) FROM %s WHERE p0 = ? AND leaf = ? AND p1 = ? AND key = ? LIMIT ? ALLOW FILTERING");
+          "SELECT key, p0, p1, leaf, text_value, dbl_value, bool_value, WRITETIME(leaf) FROM %s WHERE p0 = ? AND leaf = ? AND p1 = ? AND key = ? LIMIT ? ALLOW FILTERING");
```

Comment:
why is this changed now? any ideas?

Reply:
This is changed because now the search path and the write path use the same function for generating column names. The search path historically didn't care about the order of these columns, but for parameter binding purposes, the write side expects the columns to always be in the order `key, p0, ...pN, leaf, text_value, dbl_value, bool_value`